tests/test_core.py

```diff
@@ -1151,6 +1151,7 @@ def test_exceptions_off(self):
 
   def test_exceptions_minimal_runtime(self):
     self.set_setting('EXCEPTION_DEBUG', 1)
+    self.set_setting('EXIT_RUNTIME', 1)
```

Comment:
are there atexits that run in this test?

might also make sense to enable `test_atexit` in minimal runtime (maybe we could add a decorator to make it easier).

Reply:
All tests that check for stdout (any test that uses do_run_from_file for example) should set `EXIT_RUNTIME` so that stdout streams are flushed after main returns.

I have another change that applies `EXIT_RUNTIME` automatically to all such tests which I what motivated this change.

You are right we should modify `test_atexit` too.

Reply:
Unless the test has unflushed output, I think this change was not necessary?